frontend/src/pages/PassingTest/TestResult/styled.ts

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components';
 import { Box } from '@mui/system';
```

Comment:
change import styled please

Reply:
it's already done by @ostapdans99 in v1-173